src/styles/blocks/card.scss

```
@@ -0,0 +1,86 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px ;
+  width: $card-width;
+  background: #fff;
+  border: 1px solid #f3f3f3;
```

You create variables for color, but not for all 🥺 
```suggestion
  background: $c-whitre;
  border: 1px solid $c-someColor;
```